addon/helpers/equal.js

```diff
@@ -1,3 +1,13 @@
 export function equalHelper(params) {
-  return params[0] === params[1];
+    let len = params.length;
```

Comment:
@rwjblue Thank you very much for your advice, I really think too simple, directly with a function to get the!